### PR TITLE
docs: fixed dead link to t3.gg

### PR DESCRIPTION
**Pull request by RobRoseKnows**
Closes #<issue>

## ✅ Checklist

- [ X ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ X ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ X ] I performed a functional test on my final commit

---

## Changelog

- Fixed a deadlink 

---

## Screenshots

N/A

💯

### CODE REVIEW

File: www/src/pages/en/t3-collection.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Made a project using the T3 stack and want to share it? Add it to the list!
 | Description                                                       | Repo                                                            | Link                                                                  |
 | ----------------------------------------------------------------- | --------------------------------------------------------------- | --------------------------------------------------------------------- |
 | Create T3 Turbo - T3 Stack using Turborepo                        | [create-t3-turbo](https://github.com/t3-oss/create-t3-turbo)    | [create-t3-turbo.vercel.app](https://create-t3-turbo.vercel.app/)     |
-| Zapdos - a QnA app for streamers                                  | [t3-oss/zapdos](https://github.com/t3-oss/zapdos)               | [z.t3.gg](https://z.t3.gg)                                            |
+| Zapdos - a QnA app for streamers                                  | [t3-oss/zapdos](https://github.com/t3-oss/zapdos)               | [ask.ping.gg](https://ask.ping.gg)                                    |
```
Comment (juliusmarminge) [inline www/src/pages/en/t3-collection.md:14]: repo is moved to pingdotgg so can change that while we're at it

Comment (juliusmarminge) [reply www/src/pages/en/t3-collection.md:14]: https://github.com/pingdotgg/zapdos

Comment (RobRoseKnows) [reply www/src/pages/en/t3-collection.md]: Done.

**Review by juliusmarminge — APPROVED** [review]
thanks

**Pull request merged**